### PR TITLE
Added readme file

### PR DESCRIPTION
**Pull request by shristabaruah**
Added readme file and small changes

### CODE REVIEW

**Comment by gadgetfather** [comment]
image
Your logo is not loading

File: components/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<h1 align="center">
+<br />
+<a href="https://bloom-tronics.vercel.app/"><img src="assets\BloomUI Homepage.jpg" alt="Bloom UI" width="300"></a>
+
+<br/>
+</h1>
+
+> Simple Easy to use Component Library
+
+> Made using HTML, CSS and JavaScript, you can get fully responsive components and bloom your website with bloom UI.
```
Comment (gadgetfather) [inline components/README.md]: You can write these as features of your library

**Review by gadgetfather — CHANGES_REQUESTED** [review]
Also, your readme file is inside the components folder. You can move it into the root folder.

**Comment by RakshaPawar108** [comment]
Add the readme file at the root of your project as it is not visible in github right now. Also, the picture is not loading. Maybe it's an issue with the path of the file. Check that as well. Rest all looks good.

**Review by RakshaPawar108 — APPROVED** [review]

**Comment by shristabaruah** [comment]
> Add the readme file at the root of your project as it is not visible in github right now. Also, the picture is not loading. Maybe it's an issue with the path of the file. Check that as well. Rest all looks good.

Yes I  tried to fixed it. Can you see now.

**Comment by shristabaruah** [comment]
> Also, your readme file is inside the components folder. You can move it into the root folder.

Yes I fixed it can you see it now!

**Comment by RakshaPawar108** [comment]
> > Add the readme file at the root of your project as it is not visible in github right now. Also, the picture is not loading. Maybe it's an issue with the path of the file. Check that as well. Rest all looks good.
> 
> Yes I tried to fixed it. Can you see now.

Yes, it is visible now. Good to merge!

**Comment by shristabaruah** [comment]
> image Your logo is not loading

Is it visible now.I tried to fix it.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<h1 align="center">
+<br />
+<a href="https://bloom-tronics.vercel.app/"><img src="./assets\BloomUI Homepage.jpg" alt="Bloom UI" width="300"></a>
```
Comment (RakshaPawar108) [inline README.md]: Also here, I guess you have given the wrong link. It should be for CL and not e-commerce right?

**Review by RakshaPawar108 — CHANGES_REQUESTED** [review]

**Review by gadgetfather — APPROVED** [review]

**Review by RakshaPawar108 — APPROVED** [review]

**Pull request merged**